src/App.vue

```diff
@@ -36,6 +36,8 @@ export default {
   async created() {
     if (this.$router.history.current.path === "/") {
       this.renderRestoreSessionDialog = true;
+      const { message } = await this.$electronService.deleteSession("old");
```

Comment:
@dockabernathy01 if this isn't running under electron, it will break.  You need to put this inside an `if (this.$isElectron)` statement like you see [here](https://github.com/yatt-ai/yattie/blob/main/src/App.vue#L68).